.changeset/selfish-worms-buy.md

```diff
@@ -0,0 +1,7 @@
+---
+'hive': major
```

Comment:
we say it doesn't impact self hosting folks but this breaking change to them would mean it is breaking/not easy to upgrade?

Reply:
I am "following" semver versioning here. If you have old accounts it is technically a breaking change.